src/wrapper.ts

```diff
@@ -34,17 +34,34 @@ const NODE: unique symbol = Symbol('node');
 
 // eslint-disable-next-line @typescript-eslint/no-namespace
 export namespace Signal {
-  export let isState: (s: any) => boolean,
-    isComputed: (s: any) => boolean,
-    isWatcher: (s: any) => boolean;
+  const guards = {
```

Comment:
why define these here with fake implementations and then override them in static blocks?

Is this fallout from how namespaces work?

Would it be better to not use a namespace? (iirc, TS is trying to discourage them)